float32 z_cmd
float32 pitch_cmd
float32 roll_cmd
float32 yaw_cmd